addons/context_actions/functions/fnc_selectVehicleWeaponDialog.sqf

```diff
@@ -0,0 +1,102 @@
+#include "script_component.hpp"
```

Comment:
Again, a bunch of code is shared here too.

Reply:
I could just get ride of the dialog. It's was written first and it's only running on the parent action. What do you think?

Reply:
Yeah, maybe the dialog is not really needed as the context actions would definitely suffice.